scripts/result_checker.py

```diff
@@ -272,6 +219,7 @@ def main(master_file, all_files, tolerance, mean_tolerance, stringcolumn):
         sys.exit(1)
 
     if parsed_arguments.sort:
+        # pylint: disable=consider-using-with
```

Comment:
Careful with this. I don't think Python guarantees that all objects will be closed/deallocated at interpreter shutdown. So the temporary file might not be deleted.

Reply:
Yeah. I didn't want to take on big changes when applying the linter change but if you want, I can fix this particular issue.

Reply:
No need to fix, but please add a TODO. Otherwise someone could easily assume that someone previously verified that this was safe and correct.